app/Http/Controllers/CourseController.php

```diff
@@ -27,7 +28,8 @@ public static function index()
      */
     public function create()
     {
-        //
+        $teachers = User::where('role', 1)->select('id', 'name')->get();
```

Comment:
we need an enum for roles. consider putting this on a separate file. cc: @rubendelapena [PR #67](https://github.com/ProyectoIntegrador2018/BlueMarket/pull/67)